src/Components/encabezado/styles.css

```
@@ -0,0 +1,17 @@
+header{
+  background-color: #198ffb;
+  margin: 0;
+  padding: .5em;
+}
+
+header>h1{
```

Intenta respetar los espaciados para que el scss sea más legible
```suggestion
header > h1 {
```